.eslintignore

```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+mock
```

Comment:
I think we might want to check in the mock files as they're used in order to serve a local journal

Reply:
This is just ignoring them in the linting, only the files in the src/ folder will be linted

Reply:
ah of course 👍